README.md

```diff
@@ -22,6 +22,7 @@ the more effective the evaluation metric is in capturing the nuances of sign lan
 - ✅ [Tokenized BLEU](signwriting_evaluation/metrics/bleu.py) - BLEU score for tokenized SignWriting FSW strings.
 - ✅ [chrF](signwriting_evaluation/metrics/chrf.py) - chrF score for untokenized SignWriting FSW strings.
 - ✅ [CLIPScore](signwriting_evaluation/metrics/clipscore.py) - CLIPScore between SignWriting images. (Using the original CLIP model)
+- ✅ [SymbolDistance](signwriting_evaluation/metrics/similarity.py) - symbol distance score for SignWriting FSW strings.
```

Comment:
Your file naming scheme is a little hard to swallow. 
`similarity.py`, `similarity_metrics.md` and `test_symbol_distance.py`
I feel like if the metric is named `symbol_distance`, then the first two should be `symbol_distance.py` and `symbol_distance.md`